less/lists.less

```diff
@@ -394,15 +394,24 @@
   }
 }
 
-// + 1 more card per row than usual, max 4
+// + 1 more card per row than usual, max 7
 .documents-list-section-container, .section.associations, .view-details-associations {
   &.no-map .list-item{
-    @media (min-width: 1200px) and (max-width: @large-width) {
-      flex-basis: 32% !important;
+    @media (min-width: 2400px) {
+      flex-basis: 14% !important;
+    }
+    @media (max-width: 2400px) {
+      flex-basis: 16% !important;
+    }
+    @media (max-width: 2100px) {
+      flex-basis: 19% !important;
     }
-    @media (min-width: @large-width) {
+    @media (max-width: 1800px) {
```

Comment:
sometimes this less file have variable ... sometimes not. For me it's more readable whith direct value ... but I can use existant variable or create some new one.